Clarify namespace, clarify relation service - entity (#730)
cap-js/docs: cds/cdl.md

GET: /ViewInService(p1=5, p2=true)/Set
```

> To expose an entity, it not necessarily has to be lexically enclosed in the service definition.

Reviewer: ```suggestion
To expose an entity, it's not necessary to be lexically enclosed in the service definition.
```

> To expose an entity, it's not necessary to be lexically enclosed in the service definition.
> An entity's affiliation to a service is established using its fully qualified name.
> If the name of an entity contains the name of a service as prefix, it belongs to/is exposed by that service.

Reviewer: ```suggestion
To expose an entity, it's not necessary to be lexically enclosed in the service definition. An entity's affiliation to a service is established using its fully qualified name, so you can also use one of the following options:

- Add a namespace.
- Use the service as prefix.
```

> If the name of an entity contains the name of a service as prefix, it belongs to/is exposed by that service.

Reviewer: Does using the namespace and using the prefix have the same result? "fully qualified name" does describe the namespace example, right?

Author: Yes, using the namespace and using the prefix have the same result.
```cds
namespace foo.bar;
entity moo { };
```
creates an Entity with "fully qualified name" `foo.bar.moo`.
```cds
entity foo.bar.moo { };
```
creates an Entity with "fully qualified name" `foo.bar.moo`.
```cds
namespace foo;
entity bar.moo { };
```
creates an Entity with "fully qualified name" `foo.bar.moo`.
```cds
namespace foo;
service bar {
  entity moo { };
}
```
creates an Entity with "fully qualified name" `foo.bar.moo`, plus the service `foo.bar`.

Reviewer: Thanks!

> In the following example, all entities belong to the same service:

Reviewer: ```suggestion
In the following example, all entities belong to/are exposed by the same service:
```
